### DISCUSSION TITLE
Is a ring subscription plan required to use ring-mqtt in home assistant

### DISCUSSION THREAD

**Discussion by mazinkhu** [tsightler/ring-mqtt#743] [Support Q&A]
I just purchased a ring alarm system v2. I found out in order to arm or disarm the alarm using the app, I need to sign up for the basic service subscription plan. If I setup a home assistant server, and would like to use the ring base station as a siren and have the sensors connected to the base station. When using ring-mqtt with home assistant, do I still need to sign up for a ring basic service subscription?

**Comment by tsightler** [comment]
I'd guess almost certainly yes, but I've not tried it as I'm grandfathered into the $100/yr monitoring plan.  Whether it works or not is based on whether they actually enforce this at the API or from within the application.  If they just disable the arm/disarm functions in the app based on subscription level, then it might work, and they do implement some restrictions this way, however, they are also quite aware of these community projects, so my guess is they probably enforce it at the API level.  All I can suggest is that you test it to find out for sure.

**Comment by mazinkhu** [comment]
Thank you for the reply.
